ecr/v_func.ecr

```diff
@@ -10,6 +10,9 @@
         raise GICrystal::ObjectCollectedError.new if %gc_collected || %instance.null?
 
         %retval = %instance.as(self).{{ impl_method_name.id }}(<% call_user_method(io) %>)
+        <% if vfunc.caller_owns.full? && vfunc.return_type.object? %>
+          LibGObject.g_object_ref(%retval)
```

Comment:
I known that this doesn't compile on e.g. GskNode, GParam... that are all objects but not gobjects. However other places in generator suffer from same issue, I plan to fix that one day in all places, with proper tests. not now.